packages/core/directives/marker.ts

```diff
@@ -146,8 +146,12 @@ export class AgmMarker implements OnDestroy, OnChanges, AfterContentInit {
 
   /** @internal */
   ngOnChanges(changes: {[key: string]: SimpleChange}) {
-    if (typeof this.latitude !== 'number' || typeof this.longitude !== 'number') {
-      return;
+    if (typeof this.latitude !== 'number') {
```

Comment:
I don't we want to convert ANY value that is other than number. We should only convert strings. Than I'm ok with that change. Could you also create PRs for the other types like map or infoWindow?